src/psyh/fuckingRobot/MoveFuckingRobot.java

```diff
@@ -0,0 +1,184 @@
+package psyh.fuckingRobot;
+
+public class MoveFuckingRobot {
+
+    private static int horizontalDistance, verticalDistance;
+
+    private static int demeentionOfWolking(Direction direction) {
+        return direction == Direction.DOWN || direction == Direction.UP ? verticalDistance : horizontalDistance;
+    }
+
+    private static void wolker(FuckingRobot robot, int distance) {
+        for (; distance > 0; distance--) {
+            robot.stepForward();
+        }
+    }
+
+// Це найгірше, що зі мною траплялося за останній час. Або так код не має писатись, або я не хочу бути програмістом.
+// Софія, пробач, будь ласка, коли я це маніакально писав, а всілякого графоманства було разів у 7 більше,
+// я меньш за все думав, що комусь потім доведеться знов торувати цей нелюдяний шлях.
+    public static void moveFuckingRobot(FuckingRobot robot, int toX, int toY) {
```

Comment:
this is one of the worst ways to make the robot move
if you don't know how make this method better, I will help you. you can write me.